extension/test/e2e/klarna.spec.js

```diff
@@ -5,10 +5,12 @@ const ctpClientBuilder = require('../../src/ctp/ctp-client')
 const configBuilder = require('../../src/config/config')
 const { routes } = require('../../src/routes')
 const httpUtils = require('../../src/utils')
+const pU = require('../../src/paymentHandler/payment-utils')
```

Comment:
```suggestion
const paymentUtils = require('../../src/paymentHandler/payment-utils')
```

Reply:
This Pr is only for documentation, not for code conventions. I will resolve those comments.